src/communication/pipe.hpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef PIPE_HPP_
+#define PIPE_HPP_
+
+#include <aos/common/tools/string.hpp>
+
+#include "transport.hpp"
+
+namespace aos::zephyr::communication {
+
+class Transport : public TransportItf {
```

Comment:
Class should be named `Pipe`

Reply:
i prefer to left it as Transport and in this case will not need to use `#ifdef ` in code when we use name `Pipe` or `XenVchan`

Reply:
Done